charts/pomerium/templates/secret.yaml

```diff
@@ -13,6 +13,7 @@ type: Opaque
 data:
   cookie-secret: {{ default (randAscii 32 | b64enc | b64enc) (.Values.config.cookieSecret | b64enc )}}
   shared-secret: {{ default (randAscii 32 | b64enc | b64enc) (.Values.config.sharedSecret | b64enc )}}
+  signing-key: {{ .Values.proxy.signingKey | b64enc }}
```

Comment:
why not generate it like the rest above?

Reply:
Generating it would change default behavior. By default the value is an empty string not enabling JWT token in response headers.

Reply:
It'd be nice to have it enabled by default - I don't think that would break anything.  

To do that correctly, I think we'd move it into a TLS secret with hook annotations like the https certs.  I believe  - @desimone can you confirm - pomerium will be implicitly doing this anyway in the next version, so we might as well try to do the right thing out of the box.

@tgomas would you mind taking a stab at that?  I can always do a follow up, otherwise.

Reply:
@travisgroth That's right. If no signing key is provided, the authorize service (moved from proxy) generates one and prints out the public key.

https://github.com/pomerium/pomerium/pull/514/files#diff-d1e68c602dfe07e82a613b34bafa753bR54-R76

Reply:
Thank you for explanations and advice about hook annotation.
@travisgroth I'll look further on that way

Reply:
Now the signing key is managed as TLS secrets, that means it is generated at install and can be forced to be regenerated like TLS secrets.

However I wonder that having this JWT signing key secret being defined in `tls-secrets.yaml` and its generation being activated by `generateTLS` and `forceGenerateTLS` could be misleading. On the other hand, adding JWT dedicated file and values to manage only one signing key is probably not what we want since it make sense to (re)generate all those secrets at same time.

@travisgroth have you any idea on how to deal with that ?